src/plot.d.ts

```diff
@@ -0,0 +1,75 @@
+import type {ChannelValue} from "./channel.js";
+import type {Data, Markish} from "./mark.js";
+import type {ProjectionFactory, ProjectionImplementation, ProjectionName, ProjectionOptions} from "./projection.js";
+import type {ScaleLegendOptions, Scale, ScaleOptions} from "./scales.js";
+
+export interface PlotOptions {
```

Comment:
Suggestion: Use generics here to represent the shape of Data

```suggestion
export interface PlotOptions<T> {
  ...
  filter?: (d: T) => boolean;
  text?: (d: T) => string;
  ... 

}
```

Reply:
Different marks often have different data so it’s not always appropriate to do template the data at the Plot level. We might do this for the MarkOptions.